log2ram

```diff
@@ -10,19 +10,40 @@ isSafe () {
     [ -d $HDD_LOG/ ] || exit 1
 }
 
+remountRW() {
+    touch $HDD_LOG/$$ 2>/dev/null >/dev/null
+    RESU=$?
+    INITIAL_STATUS=ro
+    if  [ "$RESU" != "0" ] ; then
+	mount -o remount,rw ${HDD_LOG}
```

Comment:
can you fix indentation ?

Reply:
Hello,
Yes I confirm your understanding.
My use case, is for my raspberry pi on sdcard, which work on readonly filesystem and I like to use ram2log to keep & synchronize punctually log to sdcard.